Cargo.toml

```diff
@@ -5,13 +5,12 @@ authors = []
 edition = "2018"
 
 [dependencies]
-amethyst = "0.10.0"
-amethyst_derive = "0.3.0"
-amethyst_test = "0.1.0"
+amethyst = { git = "https://github.com/amethyst/amethyst.git", features = ["vulkan"]}
+#amethyst_test = "0.1.0"
 rand = "0.6.5"
 log = "0.4.6"
-amethyst-imgui = { git = "https://github.com/awpteamoose/amethyst-imgui.git", branch = "amethyst-v0.10" }
-amethyst-inspector = { git = "https://github.com/awpteamoose/amethyst-inspector.git", branch = "amethyst-v0.10" }
+#amethyst-imgui = { git = "https://github.com/awpteamoose/amethyst-imgui.git"}
```

Comment:
I've released 0.2 of amethyst-imgui which supports the latest master of amethyst.  Inspector may not support it, but that should be an easy forward port.